config/generate_lib/generic/build.sh

```diff
@@ -37,17 +37,16 @@ pushd $FW_TARGETDIR/mcu_ws >/dev/null
 		-DCMAKE_TOOLCHAIN_FILE=$TOOLCHAIN \
 		-DCMAKE_VERBOSE_MAKEFILE=ON; \
 
-    mkdir -p $FW_TARGETDIR/libmicroros; cd $FW_TARGETDIR/libmicroros; \
-	for file in $(find $FW_TARGETDIR/mcu_ws/install/lib/ -name '*.a'); do \
-		folder=$(echo $file | sed -E "s/(.+)\/(.+).a/\2/"); \
-		mkdir -p $folder; cd $folder; ar x $file; \
-		for f in *; do \
-			mv $f ../$folder-$f; \
-		done; \
-		cd ..; rm -rf $folder; \
+	mkdir -p $FW_TARGETDIR/libmicroros; cd $FW_TARGETDIR/libmicroros; \
+	echo "CREATE libmicroros.a" > $FW_TARGETDIR/libmicroros/ar_script.m; \
+	for file in $(find "$FW_TARGETDIR/install/lib/" -name '*.a'); do \
```

Comment:
```suggestion
	for file in $(find "$FW_TARGETDIR/mcu_ws/install/lib/" -name '*.a'); do \
```
Wrong path

Reply:
yeah, as I wrote earlier: this was a verbatim copy. I don't use the same folder structure, hence the discrepancy in the path.